pages/collection-list.md

```diff
@@ -0,0 +1,81 @@
+     items:
+       - Status changed by Ben Cunningham
+       - 4 days ago
+     image-source: https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Revision Needed
+     status-color: bg-secondary-light
+   - title: "3: Employee awareness"
+     items:
+       - Files Uploaded by Jamie Lee
+       - 1 hour ago
+     image-source: https://images.unsplash.com/photo-1542305983-c4100e4b8cd2?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=881&q=80
+     status: Ready for Review
+     status-color: bg-accent-warm-light
+   - title: "4: Implement measures to detect, prevent, and respond to security failures related to personal information"
+     items:
+       - Commented by Ben Cunningham
+       - 1 hour ago
+     image-source: https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Approved
+     status-color: bg-mint
+   - title: "5: Procedures in place to test the effectiveness of the safeguards referred in the previous question"
+     items:
+       - Commented by Ben Cunningham
+       - 6 hours ago
+     image-source: https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Revision Needed
+     status-color: bg-secondary-light
+   - title: "6: Notify the controller of occurrences of a breach of the privacy or security of their organization’s personal information"
+     items:
+       - Commented by Erika Spoon
+       - 1 hour ago
+     image-source: https://images.unsplash.com/photo-1597223557154-721c1cecc4b0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Completed
+     status-color: bg-accent-cool-light
+   - title: "7: Secure disposal or return of personal information when instructed by the controller or upon termination of the relationship with the controller"
+     items:
+       - Commented by Ben Cunningham
+       - 2 days ago
+     image-source: https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Approved
+     status-color: bg-mint
+   - title: "8: Compliance with instructions related to processing personal information"
+     items:
+       - Commented by Erika Spoon
+       - 30 mins ago
+     image-source: https://images.unsplash.com/photo-1597223557154-721c1cecc4b0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=880&q=80
+     status: Ready for Review
+     status-color: bg-accent-warm-light
+   - title: "9: Procedures in place to delete, update, and correct information"
+     items:
+       - Commented by Jamie Lee
```

Comment:
"commented by" language a bit awkward. either "comment by" or "commented on by..."? Or "Erika Spoon commented? Something like that.